x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/reducer.ts

```diff
@@ -222,6 +261,32 @@ export const validateAdvancedEditor = (state: State): State => {
   return state;
 };
 
+function validateMinMML(estimatedMml: string) {
+  return (mml: string) => {
+    // @ts-ignore
```

Comment:
What are these `ts-ignore`s for?

Reply:
there is a typing issue with `@elastic/numeral'`, it complains about non-existing `value`